packages/api-db-karma/src/index.ts

```diff
@@ -372,11 +372,7 @@ export class KarmaStorageAdapter implements StorageAdapter {
       define('article', all(tag(ModelTag.Article))
         .filterList((index, value) => value.field('id').equal(string(id)))
         .first()),
-      define('articleVersion', scope('article')
-        .field('versions')
-        .slice(version, 1)
-        .first()
-        .get()),
+      define('articleVersion', scope('article').field('versions').slice(version, 1).first().get()),
```

Comment:
please revert all changes in this file

Reply:
this was applied by `yarn prettier` why remove? or better exclude unmaintained from liniting?

Reply:
just remove it from the PR.